### PR TITLE
Shock Centipede PR

### PR DESCRIPTION
**Pull request by EnderMage99**
## About The Pull Request

This PR adds Shock Centipede into the game. As of the time of creating this PR, They still need sprites and a lot of their code to be fixed.

## Why It's Good For The Game

More abnos = Better

## Changelog
:cl:
add: new abno, shock centipede
/:cl:

### CODE REVIEW

**Comment by Gboster-0** [comment]
holy shit this code is so clean

File: code/modules/mob/living/simple_animal/abnormality/he/shock_centipede.dm [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+	var/IMMORTAL_COUNTDOWN_DURATION_DECISEC = 20
+
+	var/TAILATTACK_COOLDOWN_DECISEC = 80
+	var/TAILATTACK_DAMAGE = 20
+	var/TAILATTACK_DAMAGETYPE = BLACK_DAMAGE
+	var/TAILATTACK_WINDUP_DECISEC = 15
+	var/TAILATTACK_CHARGE_PER_TARGET = 3
+
+	var/TAILATTACK_RANGE = 5
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/he/shock_centipede.dm:80]: Why are these all uppercase?

**Review by LanceSmites328 — COMMENTED** [review]
Cool mechanics, let me know when it's done. Right now you have debug code in it.

**Comment by LanceSmites328** [comment]
Converting to a Draft because this has: Debug Code and Missing Sprites.

**Comment by Kitsunemitsu** [comment]
Check integration tests, it's not100% important, but do fix eventually

**Review by Coxswain-Navigator — APPROVED** [review]
It could use some more cleanup, but I won't use that an excuse to keep this out of the game any longer. I'll follow this up with another PR to address those concerns.

**Review by Coxswain-Navigator — APPROVED** [review]
Merging...

**Pull request merged**